modules/add-contextual-data/add-contextual-data-glob-selector.c

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2016 Balabit
```

Comment:
Please use the current year for the copyright.

Reply:
fixed.